scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/entrypoints/AutoGiftBox.kt

```diff
@@ -30,7 +30,7 @@ class AutoGiftBox @Inject constructor(
     override fun script(): Nothing {
         val xpOffsetX = (locations.scriptArea.find(images[Images.GoldXP]) ?: locations.scriptArea.find(images[Images.SilverXP]))
             ?.region?.center?.x
-            ?: throw Exception("Couldn't find Embers on screen. This shouldn't happen.")
+            ?: throw Exception("Couldn't find Embers on screen. Make sure that you've filtered for EXP cards.")
```

Comment:
Can you add this to KnownException.
That way the stack trace won't be shown.

Reply:
KnownException is part of `app`, this code is part of `scripts`.

Reply:
Oh right, may bad.
Do we have a ExitReason thing for GiftBox like we have for Battle and other stuff?

Reply:
We only have ExitException with the number of selected cards.

Reply:
Hmm, we don't have it yet.
If you don't mind, can you add that.

This used to be an unexpected exception until the option to receive embers was added to lottery script.